cli/up.go

```diff
@@ -0,0 +1,26 @@
+package cli
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var (
+	upCmd = &cobra.Command{
+		Use:   "up <group> [scale]",
+		Short: "Bring a group up",
+		Long:  "Submit a group, with an optional scale, and start it",
+		Run:   upRun,
+	}
+)
+
+func upRun(cmd *cobra.Command, args []string) {
+	submitRun(cmd, args)
```

Comment:
@zeisss sorry, i misunderstood, that is the current implementation.

I think if I `up`'d 4, I would expect four to be there, but I don't think it's surprising that if you submit 2 of the same group beforehand, they also get started.